objects/d3f_technique.json

```
@@ -1,18 +1,22 @@
 {
   "caption": "MITRE DEFEND™ Technique",
-  "description": "The MITRE DEFEND™ Technique object describes the leaf defensive technique ID and/or name associated to a countermeasure, as defined by <a target='_blank' href='https://d3fend.mitre.org'>D3FEND<sup>TM</sup> Matrix</a>.",
-  "extends": "_entity",
   "name": "d3f_technique",
+  "description": "The MITRE DEFEND™ Technique object describes the leaf defensive technique ID and/or name associated to a countermeasure.",
```

```suggestion
  "description": "The MITRE D3FEND™ Technique object describes the leaf defensive technique ID and/or name associated to a countermeasure.",
```
Should be "D3FEND" (not "DEFEND").